src/common/snippets/src/op/subgraph.cpp

```diff
@@ -62,16 +68,69 @@ void snippets::op::Subgraph::init_config() {
             ov::is_type<ov::op::v0::FakeQuantize>(op);
         config.m_has_type_relaxed_ops = config.m_has_type_relaxed_ops ||
             std::dynamic_pointer_cast<ov::op::TypeRelaxedBase>(op);
-        config.m_has_domain_sensitive_ops = config.m_has_domain_sensitive_ops ||
-            ov::is_type<ov::op::v1::Transpose>(op) ||
-            ov::is_type<ov::op::v1::Softmax>(op) ||
-            ov::is_type<ov::op::v8::Softmax>(op) ||
-            ov::is_type<ov::op::v0::MatMul>(op);
+        config.m_has_domain_sensitive_ops = config.m_has_domain_sensitive_ops || is_domain_sensitive_op(op);
     }
     // Domain sensitive ops are decomposed with explicit Loops. So, we should explicitly insert Loops in Subgraph if it contains these ops
     config.m_explicit_loop_insertion = config.m_has_domain_sensitive_ops;
 }
 
+auto snippets::op::Subgraph::get_estimated_buffer_count(const ov::NodeVector& ops) -> size_t {
```

Comment:
I'd suggest to think about that after release. What do you think?
Because it should the whole task to analyze the possible needed count of registers when we don't have Buffers in body yet. Moreover, it depends on Graph Coloring algorithm as well